watchface

```diff
@@ -143,16 +147,42 @@ function deployface {
     fi
 }
 
+function deleteTempFontconfig {
+    local temp_font_config="$1"
+    echo "Deleting temporary fontconfig config file \"${temp_font_config}\""
+    rm -f "${temp_font_config}"
+}
+
 function testface {
     local sourcewatchface
     if ! sourcewatchface=$(sanitizeSourceWatchface "$1") ; then
         exit 1
     fi
     echo "Testing ${sourcewatchface}"
+
+    # Create a temporary fontconfig configuration file that points to the
+    # fonts subdirectory of the watchface. This allows Qt to load the font
```

Comment:
What about watchfaces that do not have a fonts subdirectory such as `analog-nort` or `arc`?  Couldn't we just skip all of this for such cases?